packages/api/cms-api/src/mikro-orm/migrations/Migration20231206123505.ts

```diff
@@ -0,0 +1,10 @@
+import { Migration } from "@mikro-orm/migrations";
+
+export class Migration20231206123505 extends Migration {
+    async up(): Promise<void> {
+        this.addSql('drop table if exists "UserContentScopes";');
+        this.addSql('drop table if exists "UserPermission";');
```

Comment:
can we somehow detect if this table was created by application code and not delete it then?

Reply:
You're right, we can't simply delete it. One solution would be to query the migration table if the Comet migration that added the migration has been executed.

Reply:
e8bc6851bf6463e1df4a3dc9b8a3a1c9fbec0d0f